src/backend/PortabilityService.AnalysisService/Dockerfile

```diff
@@ -0,0 +1,24 @@
+### Build Stage
+FROM microsoft/aspnetcore-build:2.0.7-2.1.105 AS build
```

Comment:
I thought about that but figured that it might be better to wait for RTM then upgrade our projects to 2.1 in a separate change.  We are not using the optional value anywhere.